README.md

```diff
@@ -247,6 +247,7 @@ These variables can be used in `pulp_config.env` and `compose.yaml`:
 - `API_HOST`: hostname where pulp expects to run (default: localhost).
 - `API_PORT`: port that pulp expects to run on. This port will also get exposed on the pulp container (default: 5001).
 - `API_PROTOCOL`: can be http or https (default: http).
+- `API_ROOT`: root prefix path that is used for the pulp-cli (default: /pulp/)
```

Comment:
Maybe I didn't state my case properly. What I wanted to see is one option that reroutes pulp _and_ configures the CLI accordingly.